sdk/storage/Azure.Storage.DataMovement/MigrationGuide.md

```diff
@@ -146,6 +146,153 @@ TODO
 
 ## Migration Samples
 
+This section contains side-by-side samples of legacy vs modern library usage of various features.
+These samples are not meant to be exhaustive, but demonstrate a wide variety of uses that may need migration.
+
+- Upload
+  - Upload single file to blob storage
+  - Upload directory to blob storage
+- Download
+  - Download single blob
+  - Download blob directory
+- Service to service copy
+  - Copy blob to blob
+  - Copy share file to blob
+
+### Upload
+
+#### Upload single file to blob storage
+
+**Legacy:**
+```csharp
+// these values provided by your code
+string filePath, containerName, blobName;
+CloudBlobClient client;
+```
+```csharp
+// upload blob
+await TransferManager.UploadAsync(
+    filePath,
+    client.GetContainerReference(containerName).GetBlockBlobReference(blobName));
+```
+**Modern:**
+```csharp
+// these values provided by your code
+string filePath, blobUri;
+LocalFilesStorageResourceProvider files;
```

Comment:
can remove this line now and update below to be
`LocalFilesStorageResourceProvider.FromFile(filePath)`

Reply:
Same for the rest for `FromDirectory` as well